www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -320,7 +320,8 @@
 
     $data["start_time"] = $centreonGMT->getDate("G:i", time(), $gmt);
     $data["end_time"] = $centreonGMT->getDate("G:i", time() + $defaultDuration, $gmt);
-    $data["end"] = $centreonGMT->getDate("m/d/Y", time() + $defaultDuration, $gmt);
+    $data["start"] = strftime("%x",$centreonGMT->getDate("U", time(), $gmt));
+    $data["end"] = strftime("%x",$centreonGMT->getDate("U", time() + $defaultDuration, $gmt));
```

Comment:
Sounds like `$data["end"]` is not used at all, and was a typo in #7597, added in `AddDowntime.php` but not in `massive_downtime.php` (I then simply removed it in #7606).
`$data["start"]` seems then useless too.

Reply:
Yes, it's true.
If we don't force the date format through the backend, the datepicker javascript library will format them automatically.